server/router/src/main/java/org/infinispan/server/router/configuration/builder/RouterConfigurationBuilder.java

```diff
@@ -1,17 +1,18 @@
 package org.infinispan.server.router.configuration.builder;
 
-import org.infinispan.server.router.configuration.MultiTenantRouterConfiguration;
+import org.infinispan.server.router.configuration.RouterConfiguration;
 
 /**
  * Multi tenant router configuration builder.
  *
  * @author Sebastian Łaskawiec
  */
-public class MultiTenantRouterConfigurationBuilder implements MultiTenantConfigurationBuilderParent {
+public class RouterConfigurationBuilder implements ConfigurationBuilderParent {
```

Comment:
Do we have to worry about backwards compatibility here? cc @tristantarrant

Reply:
I think it might be a good point. However this feature is so much tied to the server that I won't expect anybody to use it directly. Even though it's bending the rules, I would just document class signature changes in the upgrading guide and leave the new names.

Reply:
This code runs in the server so don't think there's a need to keep backwards compatibility. Users don't normally have to touch this. The only users it could potentially affect is those that create an embedded server, but this is very much niche use case.